README.md

```diff
@@ -78,6 +78,13 @@ I'm open to adding more drivers, feel free to file an issue.
 
 ## To use from Kubernetes:
 
+### Using Deployment and Service
+Follow this [instruction](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/Kubernetes.md)
+
+### Using Helm
+Follow this [instruction](https://github.com/kubernetes/charts/tree/master/stable/gcloud-sqlproxy)
```

Comment:
I have added some note in the example YAML. 
I think it is better than adding in readme or description, since a lot of people just copy and paste :)